### PR TITLE
add create story API

### PR DESCRIPTION
**Pull request by drselump14**
Based on question #3, I decided to create pull request for create story wrapper

### CODE REVIEW

**Comment by drselump14** [comment]
and remove some trailing spaces on the source code also.

File: lib/pivotal-tracker-api/service.rb [hunk 1]
```diff
@@ -147,57 +154,84 @@ def stories(opts={})
...
+        end
+
+        api_url = "/projects/#{project_id}/stories/#{story_id}/comments"
+        response = PivotalAPI::Client.post(api_url, text: text)
+        json_story = JSON.parse(response, {:symbolize_names => true})
+        PivotalAPI::Comment.from_json(json_story)
+      end
+
+      private
+        def load_story_from_response(response)
```
Comment (atljeremy) [inline lib/pivotal-tracker-api/service.rb]: Maybe just name this `story_from_response`? Don't really think it needs to be prefixed with `load_`.

File: README.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ A Pivotal Tracker API gem that can be used to interface with the Pivotal Tracker
 ```ruby
 # Use your personal pivotal token
 PivotalAPI::Service.set_token A_PIVOTAL_TOKEN
+
+#then get me object
```
Comment (atljeremy) [inline README.md]: Can you add a space between `#` and `then`? Super small, I know. But would like to keep consistency with other comments. Thanks.

File: lib/pivotal-tracker-api/service.rb [hunk 3]
```diff
@@ -8,7 +8,14 @@ class << self
...
       end
-      
+
+      def get_me
+        response = PivotalAPI::Client.get("/me")
+        json_me = JSON.parse(response, {:symbolize_names => true})
+        PivotalAPI::Me.from_json(json_me)
+      end
+
+
```
Comment (atljeremy) [inline lib/pivotal-tracker-api/service.rb]: Looks like there are two empty lines here. Can you reduce to one?

**Comment by atljeremy** [comment]
Closing due to no activity.

**Pull request closed without merging**